src/app/components/harvest/pages/list/list.component.ts

```diff
@@ -44,6 +44,55 @@ class ListComponent extends PageComponent implements OnInit {
   public ngOnInit(): void {
     this.project = this.route.snapshot.data[projectKey].model;
     this.canCreateHarvestCapability = this.project.can("createHarvest").can;
+    this.filters$ = new BehaviorSubject({
```

Comment:
add a comment explaining why a whole BehaviorSubject is needed for a seemingly static filter